specification/dimensions/servicename.md

```diff
@@ -0,0 +1,30 @@
+# Service Name
+
+Service Name is an identifier assigned to a service by the provider. The Service Name is commonly used for cost reporting, analysis and allocation scenarios.
+
+The Service Name column MUST be present in the cost data. This column MUST be of type String and MUST NOT contain null values.
```

Comment:
In other specs, we used the column ID here. Do you have a preference? Either way, we should be consistent.
```suggestion
The ServiceName column MUST be present in the cost data. This column MUST be of type String and MUST NOT contain null values.
```

Reply:
100%. Let me update. I made the same comment on the charge type one PR :)